spark/src/main/scala/io/delta/sharing/spark/RemoteDeltaCDFRelation.scala

```diff
@@ -71,8 +81,9 @@ object DeltaSharingCDFReader {
       removeFiles: Seq[RemoveFile],
       schema: StructType,
       isStreaming: Boolean,
-      refresher: () => Map[String, String],
-      lastQueryTableTimestamp: Long = System.currentTimeMillis()
+      refresher: () => (Map[String, String], Long),
+      lastQueryTableTimestamp: Long,
+      minUrlExpirationTimestamp: Long
```

Comment:
Should we just call this expirationTimestamp?

Reply:
updated.